src/db_queries/partners_dashboard_stats.ts

```diff
@@ -7,6 +7,8 @@ import { constants } from "../utils/constants";
 const pool = new Conn().pool;
 
 interface NgoStats {
+  /** total number of mentors who have scheduled at least 1 course */
```

Comment:
This interface was actually created for stats related to our partner NGOs, not companies. The total number of mentors was indeed missing however. Thanks for adding it and kindly adapt the query as well so that it includes the mentors who have taught the students from our partner NGOs (they are not from the same organizations as the students so we cannot use the same `orgId` for retrieving the mentors).